src/index.html

```diff
@@ -7,6 +7,41 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__container calendar__container--start-day--wed calendar__container--month-length--28">
```

Comment:
```suggestion
    <div class="
      calendar
      calendar--start-day-wed
      calendar--month-length-28
    ">
      <div class="calendar__container">
```
Read task carefully.
Use proper modifiers naming.